src/planning/nav2_planning_tests/planner_tester.cpp

```diff
@@ -80,6 +80,9 @@ void PlannerTester::spinThread()
 void PlannerTester::loadMap(const std::string image_file_path, const std::string yaml_file_name)
```

Comment:
This function duplicates a lot of code from the nav2_map_server. I think it takes from a utility file (map_loader) in libs that duplicates the loadMapFromFile functionality. Do we want this to make that utility as part of libs or keep it maintained in nav2_map_server?  My caution is that if we are loading files other than the types here (in this case Occupancy Grids), it would require extensions in the utility functions in addition to the nav2_map_server.